singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -85,30 +89,44 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
   using Grid_t = spiner_common::Grid_t;
   using DataBox = spiner_common::DataBox;
 
+  struct TransformDataContainer {
+
+    Real lRhoOffset, lEOffset;
+    DataBox sieCold, T, dTdE;
+
+    PORTABLE_INLINE_FUNCTION
+    TransformDataContainer() = default;
+  };
+
+  using TransformDataT = TransformDataContainer;
+  using Transformer = TransformerT<TransformDataT>;
+
   struct SP5Tables {
     DataBox P, bMod, dPdRho, dPdE, dTdRho, dTdE, dEdRho;
   };
-  using STricks = table_utils::SpinerTricks<SpinerEOSDependsRhoSie>;
+  using STricks = table_utils::SpinerTricks<SpinerEOSDependsRhoSieTransformable>;
 
   SG_ADD_DEFAULT_MEAN_ATOMIC_FUNCTIONS(AZbar_)
-  SG_ADD_BASE_CLASS_USINGS(SpinerEOSDependsRhoSie);
-  PORTABLE_INLINE_FUNCTION SpinerEOSDependsRhoSie()
+  SG_ADD_BASE_CLASS_USINGS(SpinerEOSDependsRhoSieTransformable);
+  PORTABLE_INLINE_FUNCTION SpinerEOSDependsRhoSieTransformable()
       : memoryStatus_(DataStatus::Deallocated) {}
-  inline SpinerEOSDependsRhoSie(const std::string &filename, int matid, TableSplit split,
-                                bool reproducibility_mode = false);
-  inline SpinerEOSDependsRhoSie(const std::string &filename, int matid,
-                                bool reproducibility_mode = false)
-      : SpinerEOSDependsRhoSie(filename, matid, TableSplit::Total, reproducibility_mode) {
-  }
-  inline SpinerEOSDependsRhoSie(const std::string &filename,
-                                const std::string &materialName, TableSplit split,
-                                bool reproducibility_mode = false);
-  inline SpinerEOSDependsRhoSie(const std::string &filename,
-                                const std::string &materialName,
-                                bool reproducibility_mode = false)
-      : SpinerEOSDependsRhoSie(filename, materialName, TableSplit::Total,
-                               reproducibility_mode) {}
-  inline SpinerEOSDependsRhoSie GetOnDevice();
```

Comment:
looks like some trivial formatting changes here. Did you run the formatting script?